### PR TITLE
Backport "Update Installation notes"

### PR DESCRIPTION
**Pull request by stbenjam**
This change should fix Bug:

### CODE REVIEW

**Comment by jlsherrill** [comment]
ACK

File: docs/installation/index.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Katello may be installed onto a baremetal host or on a virtual guest.  The minim
 * 4 GB of memory (8 GB highly recommended)
 * The filesystem holding /var/lib/pulp needs to be large, but may vary depending on how many different Operating Systems you wish to syncronize:
   * Allocate 30 GB of space for each operating system.  Even though an operating system may not take up this much space now, this allows space for future updates that will be syncronized later.
+* The filesystem holding /var/lib/mongodb needs at least 4Gb to install, but will vary depending on how many different Operating Systems you wish to syncronize:
+  * Allocate around 40% the capacity that has been given to the /var/lib/pulp filesystem
```
Comment (ehelms) [inline docs/installation/index.md]: 40% of the capacity

File: docs/installation/index.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ Katello may be installed onto a baremetal host or on a virtual guest.  The minim
 * 4 GB of memory (8 GB highly recommended)
 * The filesystem holding /var/lib/pulp needs to be large, but may vary depending on how many different Operating Systems you wish to syncronize:
   * Allocate 30 GB of space for each operating system.  Even though an operating system may not take up this much space now, this allows space for future updates that will be syncronized later.
+* The filesystem holding /var/lib/mongodb needs at least 4Gb to install, but will vary depending on how many different Operating Systems you wish to syncronize:
```
Comment (ehelms) [inline docs/installation/index.md]: s/Gb/GB

File: docs/installation/index.md (hunk 1 above)
Comment (duncaninnes) [inline docs/installation/index.md]: The 40% figure is based on only my own own experience.  If there's a better estimate, happy to change to that.

**Pull request merged**